README.md

```diff
@@ -50,12 +50,16 @@ When rendered the function will return a formed HTML with `<ul>` and the links o
 Simply copy the `user/plugins/breadcrumbs/anchors.yaml` into `user/config/plugins/anchors.yaml` and make your modifications.
 
     enabled: true                 # enable or disable the plugin
-    active: true                  # active by default, if false then you must activate per-page
+    active: false                 # active by default, if false then you must activate per-page
```

Comment:
Probably this causes the plugin to be deactivated by default, and is not backwards compatible.  Hence when I 'updated' my test site to use this plugin, the previously working anchors stopped working.